### ISSUE TITLE
Pull down more training data from brain-map.org

### ISSUE THREAD

**Issue by JohnTigue** [reconstrue/brightfield_neuron_reconstruction#67]
See #60 for a related, earlier idea that is now reinforced by this issue.

Currently it seems The Allen is gather data on about 6,000/y but are only reconstructing the skeletons for about 500/y. That's their main problem. Nonetheless, that means there are hundreds of more specimens _with_ semi-automatically generated skeletons. That's tons more training data.

Also, this and #60 are all the more evidence that adding a mechanism to allensdk that pulls down image stacks (via their RMA interface) is a valuable PR to send back upstream.

**Comment by JohnTigue** [comment]
So, tons of training data for researchers. If this project has code which takes the raw challenge dataset and using specimens_manifest.json presents that via a module with the same function interfaces as the allensdk, then that's how this is normalized: there are specimens some of which have SWC, some don't. A challenge is then simply using the brain-map.org data (wherein some set of specimens without SWCs are used as the test set).
